core/src/player/state.rs

```diff
@@ -47,6 +47,7 @@ pub fn install(session: &mut GameSession) {
     add_state_module!(session, midair);
     add_state_module!(session, walk);
     add_state_module!(session, dead);
+    add_state_module!(session, incapacitaded);
```

Comment:
_( Read comment below about moving the state transition function first. )_

Since we don't need a `player_state_transition` system in the `incapacitated` state, don't use the module here and just register the state handler system:

```suggestion
    session
        .stages
        .add_system_to_stage(CoreStage::PreUpdate, incapacitated::handle_player_state);
```